Tiltfile

```diff
@@ -8,67 +15,83 @@ def binary():
 
 local_resource('recompile', binary(), deps=['cmd', 'gateway.go', 'kubernetes.go', 'setup.go', 'apex.go'])
 
-docker_build_with_restart(IMG, '.', 
-    dockerfile='tilt.Dockerfile', 
+
+docker_build_with_restart(IMG, '.',
+    dockerfile='tilt.Dockerfile',
     entrypoint=['/coredns'], 
-    only=['./coredns'], 
     live_update=[
         sync('./coredns', '/coredns'),
         ]
-    )
+)
+
 
+# if using mac m1+, you'll need to use colima
+#nerdctl_build(IMG, '.',
+#    dockerfile='tilt.Dockerfile', 
+#    entrypoint=['/coredns'], 
+#    live_update=[
+#        sync('./coredns', '/coredns'),
+#        ]
+#)
+
+# Cilium CNI
+helm_remote('cilium',
+            version="1.15.0-pre.1",
```

Comment:
is there any reason not to use a stable Cilium version. L2 announcement should be in 1.14.X

Reply:
yes actually, because 1.15.0-pre.1/2 support the latest gateway api crds where as the stable doesn't

Reply:
ah, I see. that's fine then. just need to bump it to a stable once  1.15 is out